build/configs/rtl8730e/loadable_apps_16M/defconfig

```diff
@@ -420,8 +425,8 @@ CONFIG_SIG_SIGWORK=17
 #
 # POSIX Message Queue Options
 #
-CONFIG_PREALLOC_MQ_MSGS=4
-CONFIG_MQ_MAXMSGSIZE=600
+CONFIG_PREALLOC_MQ_MSGS=32
```

Comment:
Please confirm this configuration value.

Reply:
These CONFIGs have been recently updated in private with a synchronization PR.
I will check all updates and make corresponding changes here.
Thankyou